README.md

```diff
@@ -20,11 +21,25 @@ terra
 
 ### Requirements
 1. Java 11
-2. Docker 20.10.2 *
+2. Docker 20.10.2 (Must be running)
 
-* Other Docker versions may also work, but this is the only one tested so far.
-
-Make sure Docker is running.
+#### Authentication
+1. Register a Terra account with the [dev instance](https://bvdp-saturn-dev.appspot.com/#) 
+to go through authentication. I think this needs to be a non-Google/Verily corporate account.
+2. Keep going through the warning login screen ("! Google hasn't verified this app" -> click 
+"Advanced" -> "Go to ... (unsafe)") that shows up since the CLI is not a Google-verified app yet.
```

Comment:
Is this after running `terra auth login`? little unclear following the instructions for registering with the app terra dev instance.

Reply:
Yes. Re-worded a little.